app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -370,10 +370,10 @@ public void showBadImagePopup(Integer errorCode) {
         String errorMessageForResult = getErrorMessageForResult(getContext(), errorCode);
         if (!StringUtils.isBlank(errorMessageForResult)) {
             DialogUtil.showAlertDialog(getActivity(),
-                    getString(R.string.warning),
+                    getString(R.string.duplicate_image_found),
                     errorMessageForResult,
                     getString(R.string.cancel),
-                    getString(R.string.upload_image),
+                    getString(R.string.upload_proceed),
```

Comment:
The question in this dialog would be "Do you still want to upload this picture?". So, using "Upload" would be more appropriate for the button.